### ISSUE TITLE
[askbot] Exceeded 30 redirects

### ISSUE THREAD

**Issue by zhquan** [grimoirelab/perceval#181]
When perceval try to collect the question on http://ask.projectatomic.io/ we have the following error
```
2017-10-16 10:45:39,067 - grimoire_elk.arthur - ERROR - Error feeding ocean from askbot (http://ask.projectatomic.io): Exceeded 30 redirects.
Traceback (most recent call last):
  File "./grimoire_elk/arthur.py", line 130, in feed_backend
    ocean_backend.feed()
  File "./grimoire_elk/ocean/elastic.py", line 201, in feed
    for item in items:
  File "/usr/local/lib/python3.4/dist-packages/perceval-0.9.1-py3.4.egg/perceval/backend.py", line 360, in decorator
    for data in func(self, *args, **kwargs):
  File "/usr/local/lib/python3.4/dist-packages/perceval-0.9.1-py3.4.egg/perceval/backends/core/askbot.py", line 96, in fetch
    html_question = self.__fetch_question(question)
  File "/usr/local/lib/python3.4/dist-packages/perceval-0.9.1-py3.4.egg/perceval/backends/core/askbot.py", line 125, in __fetch_question
    html_question = self.client.get_html_question(question['id'], npages)
  File "/usr/local/lib/python3.4/dist-packages/perceval-0.9.1-py3.4.egg/perceval/backends/core/askbot.py", line 278, in get_html_question
    response = self.__call(path, params)
  File "/usr/local/lib/python3.4/dist-packages/perceval-0.9.1-py3.4.egg/perceval/backends/core/askbot.py", line 318, in __call
    req = requests.get(url, params=params, headers=headers)
  File "/usr/local/lib/python3.4/dist-packages/requests/api.py", line 72, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/requests/sessions.py", line 502, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.4/dist-packages/requests/sessions.py", line 634, in send
    history = [resp for resp in gen] if allow_redirects else []
  File "/usr/local/lib/python3.4/dist-packages/requests/sessions.py", line 634, in <listcomp>
    history = [resp for resp in gen] if allow_redirects else []
  File "/usr/local/lib/python3.4/dist-packages/requests/sessions.py", line 134, in resolve_redirects
    raise TooManyRedirects('Exceeded %s redirects.' % self.max_redirects, response=resp)
requests.exceptions.TooManyRedirects: Exceeded 30 redirects.
2017-10-16 10:58:06,201 - grimoire_elk.arthur - ERROR - Error feeding ocean from askbot (http://ask.projectatomic.io): Exceeded 30 redirects.
```
On http://ask.projectatomic.io/ we have two questions without a topic, so maybe that is the problem

**Comment by albertinisg** [comment]
To reproduce:
```bash
$ perceval -g askbot "http://ask.projectatomic.io" --from-date 2017-10-12
...
[2017-10-16 15:44:44,927 - requests.packages.urllib3.connectionpool - INFO] - Starting new HTTP connection (1): ask.projectatomic.io
[2017-10-16 15:44:45,426 - requests.packages.urllib3.connectionpool - DEBUG] - "GET /question/4392?page=1&sort=votes HTTP/1.1" 302 20
[2017-10-16 15:44:45,428 - requests.packages.urllib3.connectionpool - INFO] - Resetting dropped connection: ask.projectatomic.io
[2017-10-16 15:44:45,668 - requests.packages.urllib3.connectionpool - DEBUG] - "GET /en/question/4392?page=1&sort=votes HTTP/1.1" 301 20
[2017-10-16 15:44:45,669 - requests.packages.urllib3.connectionpool - INFO] - Resetting dropped connection: ask.projectatomic.io
[2017-10-16 15:44:45,954 - requests.packages.urllib3.connectionpool - DEBUG] - "GET /en/question/4392/?page=1&sort=votes HTTP/1.1" 302 20
[2017-10-16 15:44:45,956 - requests.packages.urllib3.connectionpool - INFO] - Resetting dropped connection: ask.projectatomic.io
...
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/perceval-0.9.1-py3.5.egg/perceval/backend.py", line 310, in run
    for item in items:
  File "/usr/local/lib/python3.5/dist-packages/perceval-0.9.1-py3.5.egg/perceval/backend.py", line 360, in decorator
    for data in func(self, *args, **kwargs):
  File "/usr/local/lib/python3.5/dist-packages/perceval-0.9.1-py3.5.egg/perceval/backends/core/askbot.py", line 96, in fetch
    html_question = self.__fetch_question(question)
  File "/usr/local/lib/python3.5/dist-packages/perceval-0.9.1-py3.5.egg/perceval/backends/core/askbot.py", line 125, in __fetch_question
    html_question = self.client.get_html_question(question['id'], npages)
  File "/usr/local/lib/python3.5/dist-packages/perceval-0.9.1-py3.5.egg/perceval/backends/core/askbot.py", line 278, in get_html_question
    response = self.__call(path, params)
  File "/usr/local/lib/python3.5/dist-packages/perceval-0.9.1-py3.5.egg/perceval/backends/core/askbot.py", line 318, in __call
    req = requests.get(url, params=params, headers=headers)
  File "/usr/local/lib/python3.5/dist-packages/requests/api.py", line 69, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/requests/api.py", line 50, in request
    response = session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/requests/sessions.py", line 465, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.5/dist-packages/requests/sessions.py", line 594, in send
    history = [resp for resp in gen] if allow_redirects else []
  File "/usr/local/lib/python3.5/dist-packages/requests/sessions.py", line 594, in <listcomp>
    history = [resp for resp in gen] if allow_redirects else []
  File "/usr/local/lib/python3.5/dist-packages/requests/sessions.py", line 114, in resolve_redirects
    raise TooManyRedirects('Exceeded %s redirects.' % self.max_redirects)
requests.exceptions.TooManyRedirects: Exceeded 30 redirects.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/perceval", line 4, in <module>
    __import__('pkg_resources').run_script('perceval==0.9.1', 'perceval')
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 746, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 1501, in run_script
    exec(code, namespace, namespace)
  File "/usr/local/lib/python3.5/dist-packages/perceval-0.9.1-py3.5.egg/EGG-INFO/scripts/perceval", line 193, in <module>
    main()
  File "/usr/local/lib/python3.5/dist-packages/perceval-0.9.1-py3.5.egg/EGG-INFO/scripts/perceval", line 111, in main
    cmd.run()
  File "/usr/local/lib/python3.5/dist-packages/perceval-0.9.1-py3.5.egg/perceval/backend.py", line 315, in run
    raise RuntimeError(str(e))
RuntimeError: Exceeded 30 redirects.
```

**Comment by sduenas** [comment]
Fixed by #182
